src/scripts/create_xglue_es.py

```diff
@@ -0,0 +1,161 @@
+"""Create the Spanish XGLUE NER dataset from local files."""
```

Comment:
Just had a look at what dataset XGLUE-NER actually is, and it [turns out](https://arxiv.org/abs/2004.01401) that it is just the Spanish part of CoNLL-2002. We already use the Dutch part of CoNLL-2002, so you can substitute this script with a copy of the `create_conll_nl.py` script, and generally rename the dataset "conll-es", to be consistent with the other CoNLL NER datasets (`conll-nl` and `conll-en`).